extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -167,7 +167,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
       }
       const senderEmail = this.getSenderEmail(emailContainer);
       const isOutgoing = !!this.sendAs[senderEmail];
-      const replacementXssSafe = XssSafeFactory.replaceRenderableMsgBlocks(this.factory, emailContainer.innerText, this.determineMsgId(emailContainer), senderEmail, isOutgoing);
+      const replacementXssSafe = XssSafeFactory.replaceRenderableMsgBlocks(this.factory, emailContainer.innerText.replace('\n\n\n','\n\n'), this.determineMsgId(emailContainer), senderEmail, isOutgoing);
```

Comment:
This will cause more problems then it will fix. You shouldn't be working on signature verification, it's not trivial.